src/main/java/org/folio/invoices/utils/HelperUtils.java

```diff
@@ -231,4 +242,83 @@ private static boolean isFieldNotEmpty(Object newObject, Object existedObject, S
   private static boolean isFieldChanged(Object newObject, Object existedObject, String field) throws IllegalAccessException {
     return !EqualsBuilder.reflectionEquals(FieldUtils.readDeclaredField(newObject, field, true), FieldUtils.readDeclaredField(existedObject, field, true), true, existedObject.getClass(), true);
   }
+
+  /**
+   * Transform list of id's to CQL query using 'or' operation
+   * @param ids list of id's
+   * @return String representing CQL query to get records by id's
+   */
+  public static String convertIdsToCqlQuery(List<String> ids) {
+    return StreamEx.of(ids).map(id -> "id==" + id).joining(" or ");
+  }
+
+  /**
+   * Wait for all requests completion and collect all resulting objects. In case any failed, complete resulting future with the exception
+   * @param futures list of futures and each produces resulting object on completion
+   * @param <T> resulting type
```

Comment:
This param does not exist. Can be removed.

Reply:
https://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#@param
Tried to make the doc comment more clear.